R/shiny.R

```diff
@@ -1314,6 +1323,8 @@ ShinySession <- R6Class(
       # bootstrapTheme, (2) re-executes any registered theme dependencies, and
       # (3) sends the resulting dependencies to the client.
 
+      private$ensureCurrentThemeDependency()
+
       # Note that this will automatically scope to the session.
       shinyOptions(bootstrapTheme = theme)
```

Comment:
```suggestion
      # Note that this will automatically scope to the session.
      setCurrentTheme()
      
      private$ensureCurrentThemeDependency()
```

Reply:
I would also remove the whitespace below so the `currentThemeDependency` logic is near each other